packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/quanticGeneratedAnswer.js

```diff
@@ -197,6 +225,11 @@ export default class QuanticGeneratedAnswer extends LightningElement {
     this.generatedAnswer.retry();
   }
 
+  handleGeneratedAnswerRephrase(event) {
+    event.stopPropagation();
+    this.generatedAnswer.rephrase({answerStyle: event?.detail});
```

Comment:
Here the event is coming from our internal component so we are sure that we are not sending null, but if we accidentally do I think it would be better to let Headless throw so it would be easy to locate the issue.